examples/scripts/counter.js

```diff
@@ -25,19 +25,19 @@
 let count;
 let lastChange;
 WoT.produce({
-    title: "counter",
+    title: "Counter",
     titles: {
-        en: "counter",
-        de: "zähler",
+        en: "Counter",
+        de: "Zähler",
         it: "Contatore",
     },
-    description: "counter example Thing",
+    description: "Counter example Thing",
     descriptions: {
-        en: "counter example Thing",
+        en: "Counter example Thing",
         de: "Zähler Beispiel Ding",
         it: "Contatore Esempio",
     },
-    support: "https://github.com/eclipse/thingweb.node-wot/",
+    support: "git://github.com/eclipse/thingweb.node-wot.git",
```

Comment:
Why do you want to change the support link which can be viewed in a browser to a git link that does **not** render in a browser and does not give any support?

Reply:
The one submitted to wot-testing had this, probably due to being older than the master branch here. I will fix it